src/HttpRequest.h

```diff
@@ -254,5 +254,9 @@ class ConnStateData;
  */
 void UpdateRequestNotes(ConnStateData *csd, HttpRequest &request, NotePairs const &notes);
 
+/// \returns a pointer to known *_port address where the request was accepted
+/// \returns nil for nil requests and when the port address info was not available
+const Ip::Address *FindListeningPortAddress(const HttpRequest *request, const AccessLogEntry *al);
```

Comment:
Please remove the names of those two parameters on this line. It is obvious from the type what they are and avoids complications from the doxygen documentation tool.

Reply:
Done